### DISCUSSION TITLE
validation messages shows some werid messages with the rules lt lte and all such rules

### DISCUSSION THREAD

**Discussion by hibasalem** [vueform/vueform#357] [Questions]
when trying to use any validation rule that contains compression of the values weird messages are showing to the end users when the filed we are comparing to is not filled  (NAN , Invalid Date)

![image](https://github.com/user-attachments/assets/814c74d4-84ee-4e9f-9c6e-a3357d2a2ff0)
![image](https://github.com/user-attachments/assets/003f3785-bbaf-402d-914a-69e99569fdfd)

expected result  : 
all the messages should be readable **without adding custom messages each time** 

this includes the following : 
- after (date) 
- After Or Equal (Date) 
- Before (Date)
- Before Or Equal (Date)
- Greater Than
- Greater Than Or Equal
- Less Than
- Less Than Or Equal

schmea 
```
"min":{
      "fieldMode":"schema",
      "type":"date",
      "label":"From",
      "placeholder":"Select registration date from",
      "floating":false,
      'rules': ['required', 'before_or_equal:max'],
      'columns': {
              'container': 6,
              'label': 12,
              'wrapper': 12,
      },
    },
"max":{
      "fieldMode":"schema",
      "type":"date",
      "label":"To",
      "placeholder":"Select registration date to",
      "floating":false,
      'rules': ['required', 'after_or_equal:min'],
      'columns': {
        'container': 6,
        'label': 12,
        'wrapper': 12,
      },
}
```

```
"min": {
  "type": "text",
  "inputType": "number",
  "floating": false,
  "fieldName": "Min",
  "placeholder": "Min",
  "columns": {
    "container": 6
  },
  "rules": [
    "min:0",
    "max:30",
    "required",
    "integer",
    "lte:container.max"
  ]
},
"max": {
  "type": "text",
  "inputType": "number",
  "floating": false,
  "fieldName": "Max",
  "placeholder": "Max",
  "columns": {
    "container": 6
  },
  "rules": [
    "min:0",
    "max:30",
    "required",
    "integer",
    "gte:container.min"
  ]
}
```

**Comment by adamberecz** [comment]
I would recommend using [conditional rules](https://vueform.com/docs/validating-elements#conditional-rules) or [custom error messages](https://vueform.com/docs/validating-elements#custom-errors-and-messages) for such cases.
